api/lib/accounts.js

```diff
@@ -34,6 +34,25 @@ const registerUser = async (uid) => {
   }
 }
 
+const sessionCache = {}
```

Comment:
will this be persisted?

Reply:
No, but it doesn't need to be as it just caches the db request. On a restart the first query for a session will just go to the db.